File: v3/producers/send.js
#!/usr/bin/env node

var amqp = require('amqplib/callback_api');

const RABBITMQ_HOST = process.env.RABBITMQ_HOST || 'localhost';
const RABBITMQ_PORT = process.env.RABBITMQ_PORT || '5672';
const MENSAJE = process.env.MENSAJE || 'Hola mundo!';

console.log(`RabbitMQ Host: ${RABBITMQ_HOST}`);
console.log(`RabbitMQ Port: ${RABBITMQ_PORT}`);
console.log(`Mensaje: ${MENSAJE}`);

amqp.connect(`amqp://${RABBITMQ_HOST}:${RABBITMQ_PORT}`, function(error0, connection) {
    if (error0) {
        throw error0;
    }
    connection.createChannel(function(error1, channel) {
        if (error1) {
            throw error1;
        }

        var queue = 'hello';

        channel.assertQueue(queue, {
            durable: false
        });

        setInterval(() => {
            channel.sendToQueue(queue, Buffer.from(MENSAJE));
            console.log(" [x] ENVIADO: %s", MENSAJE);
        }, 3500);
        
    });
});

File: v3/producers/send.py
#!/usr/bin/env python
import pika
import time
import os

RABBITMQ_HOST = os.environ.get('RABBITMQ_HOST') if os.environ.get('RABBITMQ_HOST') != None else 'localhost' 
RABBITMQ_PORT = int(os.environ.get('RABBITMQ_PORT')) if os.environ.get('RABBITMQ_PORT') != None else 5672
MENSAJE = os.environ.get('MENSAJE') if os.environ.get('MENSAJE') != None else 'Hola Mundo'

print("RABBITMQ_HOST: ", RABBITMQ_HOST)
print("RABBITMQ_PORT: ", RABBITMQ_PORT)
print("MENSAJE: ", MENSAJE)

connection = pika.BlockingConnection(pika.ConnectionParameters(host = RABBITMQ_HOST, port = RABBITMQ_PORT))
channel = connection.channel()
channel.queue_declare(queue='hello')

while True:
    channel.basic_publish(exchange='',
                      routing_key='hello',
                      body=MENSAJE)
    print(" [x] ENVIADO: " + MENSAJE, flush=True)
    time.sleep(4)